leaderboard.js

```diff
@@ -335,6 +335,95 @@ function emptyLeaderboard(msg) {
 
 }
 
+function addUser(message) {
+  const authorId = message.author.id;
+  const info = message.content.replace(/!addscore +<@!?(\d+)> *(\d+)?/g, '$1-$2').split('-');
```

Comment:
Can  you add a comment explaining what this does?

Reply:
Just added the comment: It is just the get the data from the message as userId-score